sdk/go/http/testdata/http-tinygo/spin.toml

```diff
@@ -10,3 +10,5 @@ id = "http-tinygo-test"
 source = "main.wasm"
 [component.trigger]
 route = "/hello/..."
+[component.build]
+command = "tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go"
```

Comment:
I am somewhat starting to reconsider `-no-debug` as the default — I think the context we get out of having debug symbols might be worth the extra binary size.


That being said, this does not block this PR. If we decide on it, a follow-up PR should change these across the examples and eventually docs.